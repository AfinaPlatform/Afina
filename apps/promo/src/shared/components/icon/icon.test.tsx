import { render } from "@testing-library/react";
import Icon from "./icon";

describe("shared/icon", () => {
  it("render div html tag", () => {
    const { container } = render(<Icon>Here icon</Icon>);
    expect(container.querySelector("div")?.tagName).toBe("DIV");
  });

  it("snapshot", () => {
    const { asFragment } = render(<Icon>icon</Icon>);
    expect(asFragment()).toMatchSnapshot();
  });
});
