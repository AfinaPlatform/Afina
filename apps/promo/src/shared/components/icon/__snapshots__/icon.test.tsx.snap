// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shared/icon snapshot 1`] = `
<DocumentFragment>
  <div
    class="root"
  >
    icon
  </div>
</DocumentFragment>
`;
